admin/acceptance.py

```diff
@@ -143,12 +145,66 @@ def stop_cluster(reactor):
 ]
 
 
-@implementer(INode)
-@attributes(['address', 'distribution'], apply_immutable=True)
-class VagrantNode(object):
+@implementer(IClusterRunner)
+class ManagedRunner(object):
     """
-    Node run using VagrantRunner
+    An ``IClusterRunner`` implementation that doesn't start or stop nodes but
+    only gives out access to nodes that are already running and managed by
+    someone else.
     """
+    def __init__(self, node_addresses, distribution):
+        self._nodes = pvector(
+            ManagedNode(address=address, distribution=distribution)
+            for address in node_addresses
+        )
+
+    def start_cluster(self, reactor):
+        """
+        Don't start any nodes.  Give back the addresses of the configured,
+        already-started nodes.
+        """
+        return configured_cluster_for_nodes(reactor, self._nodes)
+
+    def stop_cluster(self, reactor):
+        """
+        Don't stop any nodes.
+        """
+        return succeed(None)
+
+
+def configured_cluster_for_nodes(reactor, nodes):
+    """
+    Get a ``Cluster`` with Flocker services running on the right nodes.
+
+    Generate new certificates for the services.
+
+    :param reactor: The reactor.
+    :param nodes: The ``ManagedNode``s on which to operate.
+    :returns: A ``Deferred`` which fires with ``Cluster`` when it is
+        configured.
+    """
+    certificates_path = FilePath(mkdtemp())
```

Comment:
This will be mostly ripped out in FLOC-1932 anyway; Vagrant does not need to `configure_cluster` or generate certs once FLOC-1932 is done, so it's only temporarily duplicative.